music21/musicxml/xmlToM21.py

```diff
@@ -5833,9 +5834,12 @@ def updateVoiceInformation(self):
                 # additional time < 1 sec per ten million ops.
 
         if len(self.voiceIndices) > 1:
-            for vIndex in sorted(self.voiceIndices):
+            vSequence: int = 0
+            for vId in sorted(self.voiceIndices):
+                vSequence += 1
                 v = stream.Voice()
-                v.id = vIndex  # TODO: should use a separate voiceId or something in Voice.
+                v.id = vId  # parsed from XML document
```

Comment:
This one-line change could be its own PR discussed separately, maybe. Rather than "correct" whatever we get in the `<voice>` tags, just accept whatever we get.